### PR TITLE
Fix i18n for the new Proposal model

### PR DESCRIPTION
**Pull request by bkutil**

### CODE REVIEW

File: crowbar_framework/config/locales/crowbar/en.yml [hunk 1]
```diff
@@ -267,11 +267,12 @@ en:
     service:
       name_exists: 'Name already exists. Please choose another.'
       too_short: 'Invalid name.  Name must have some characters.'
-      illegal_chars: 'Invalid service name %{name}. Please limit name to [A-z0-9_] without spaces.'
-      illegal_name: 'Illegal proposal name %{name}. Please choose another proposal name.'
+      illegal_chars: 'Please limit name to [A-z0-9_] without spaces.'
+      illegal_name: 'Please choose another proposal name.'
```
Comment (vuntz) [inline crowbar_framework/config/locales/crowbar/en.yml]: Might be worth clarifying why, as this is totally unclear from this message the name is forbidden.

**Comment by vuntz** [comment]
Travis fails :/

**Comment by bkutil** [comment]
@vuntz I addressed your comments and fixed tests.

**Comment by vuntz** [comment]
+1

**Comment by tboerger** [comment]
:+1:

**Pull request merged**